### PR TITLE
Feature#4/player

### PR DESCRIPTION
**Pull request by javs1**
# Description
What did you implemented/Why did you implemented this:
 - View of the video player
 - Recomendations for the next video to play
 - Navbar for the user
 
 Why: 
- The view of the video player is used to view old google meet calls if the user uploaded it
- Navbar is used to make easier navegation in the page
 
  
  ## Screenshots 
image

### CODE REVIEW

File: EasyLearn/app/helpers/player_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module PlayerHelper
```
Comment (xFerchoVJ) [inline EasyLearn/app/helpers/player_helper.rb]: If you don't need this file, you can delete it

File: EasyLearn/app/assets/stylesheets/player.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500&display=swap');
+
+nav {
+  background-color:#282828;
```
Comment (xFerchoVJ) [inline EasyLearn/app/assets/stylesheets/player.css]: I recommend you sort your properties of your styles alphabetically, It is considered good practice :smile:

File: EasyLearn/app/assets/stylesheets/player.css [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+.extra-video {
+  margin-right: 10px;
+}
+
+.user-option {
+  display: flex;
+  align-items: center;
+}
```
Comment (xFerchoVJ) [inline EasyLearn/app/assets/stylesheets/player.css]: Add a blank line

File: EasyLearn/app/views/player/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,54 @@
+<%= stylesheet_link_tag "player", "data-turbo-track": "reload" %>
+<body>
```
Comment (xFerchoVJ) [inline EasyLearn/app/views/player/index.html.erb]: It is not necessary to use the body tag, this tag along with the head and html tags are located in the application.html.erb file in the layouts folder

**Review by xFerchoVJ — CHANGES_REQUESTED** [review]
I left you some comments, if you have questions you can send me a message for slack :smile:

File: EasyLearn/app/helpers/player_helper.rb (hunk 1 above)
Comment (javs1) [reply EasyLearn/app/helpers/player_helper.rb]: Deleted

File: EasyLearn/app/assets/stylesheets/player.css (hunk 3 above)
Comment (javs1) [reply EasyLearn/app/assets/stylesheets/player.css]: Added

File: EasyLearn/app/views/player/index.html.erb (hunk 4 above)
Comment (javs1) [reply EasyLearn/app/views/player/index.html.erb]: Deleted body tag

File: EasyLearn/app/assets/stylesheets/player.css (hunk 2 above)
Comment (javs1) [reply EasyLearn/app/assets/stylesheets/player.css]: Sorted

**Review by LuisGomez89 — APPROVED** [review]

File: EasyLearn/app/assets/stylesheets/player.css (hunk 2 above)
Comment (JuanequeX) [reply EasyLearn/app/assets/stylesheets/player.css]: I do not see that it is sorted

File: EasyLearn/app/views/player/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,53 @@
+<%= stylesheet_link_tag "player", "data-turbo-track": "reload" %>
+
+  <nav>  
+    <h3 >Logo</h3>
+    <div class="menu">
+      <p>My events</p>
+      <div class="user-option">
+        <p class="username">username</p>
+        <%= image_tag("dropdown.png", :alt => "rss feed", :class => "profile-picture") %>
```
Comment (JuanequeX) [inline EasyLearn/app/views/player/index.html.erb]: I recommend use svg images, and create an `helper` for show it and render your images in your views

File: EasyLearn/app/views/player/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+				<div class="extra-video">
+					<video src="" controls> </video>
+					<div class="user">
+						<%= image_tag("user.png", :alt => "rss feed", :class => "profile-picture") %>
+						<p>Pedro Sanchez</p>
+					</div>
+				</div>
+    	</div>  
+  	</section>
+  </div>
```
Comment (JuanequeX) [inline EasyLearn/app/views/player/index.html.erb]: Review what is the reason that because this block show without correct indentation

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: EasyLearn/app/views/player/index.html.erb (hunk 5 above)
Comment (YilKanda) [inline EasyLearn/app/views/player/index.html.erb]:
```suggestion
        <%= image_tag("dropdown.png", alt: "rss feed", class: "profile-picture") %>
```

File: EasyLearn/app/views/player/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+        <p class="username">username</p>
+        <%= image_tag("dropdown.png", :alt => "rss feed", :class => "profile-picture") %>
+      </div>  
+    </div>
+  </nav>
+  <div class="content">
+    <video  controls class="main-video"></video>
+    <h3 class="title">Pilas y Colas en C</h3>
+    <div class="user">
+      <%= image_tag("user.png", :alt => "rss feed", :class => "profile-picture") %>
```
Comment (YilKanda) [inline EasyLearn/app/views/player/index.html.erb]:
```suggestion
      <%= image_tag("user.png", alt: "rss feed", class: "profile-picture") %>
```

File: EasyLearn/app/views/player/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+      <p>Diego Steve</p>
+    </div>
+    <section>
+      <p class="description">Other content by the owner that may interest you</p>
+      <div class="video-suggested">
+        <div class="extra-video">
+          <video src="" controls> </video>
+          <div class="user">
+            <%= image_tag("user.png", :alt => "rss feed", :class => "profile-picture") %>
```
Comment (YilKanda) [inline EasyLearn/app/views/player/index.html.erb]:
```suggestion
            <%= image_tag("user.png", alt: "rss feed", class: "profile-picture") %>
```

File: EasyLearn/app/views/player/index.html.erb [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+            <p>Mildred Nataly</p>
+          </div>
+      	</div>  
+				<div class="extra-video">
+					<video src="" controls> </video>
+					<div class="user">
+						<%= image_tag("user.png", :alt => "rss feed", :class => "profile-picture") %>
```
Comment (YilKanda) [inline EasyLearn/app/views/player/index.html.erb]:
```suggestion
						<%= image_tag("user.png", alt: "rss feed", class: "profile-picture") %>
```

File: EasyLearn/app/views/player/index.html.erb [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+						<p>Mildred Nataly</p>
+					</div>
+				</div>
+				<div class="extra-video">
+					<video src="" controls> </video>
+					<div class="user">
+						<%= image_tag("user.png", :alt => "rss feed", :class => "profile-picture") %>
```
Comment (YilKanda) [inline EasyLearn/app/views/player/index.html.erb]:
```suggestion
						<%= image_tag("user.png", alt: "rss feed", class: "profile-picture") %>
```

File: EasyLearn/app/views/player/index.html.erb [hunk 11]
```diff
@@ -0,0 +1,53 @@
...
+						<p>El bicho siuu</p>
+					</div>
+				</div>
+				<div class="extra-video">
+					<video src="" controls> </video>
+					<div class="user">
+						<%= image_tag("user.png", :alt => "rss feed", :class => "profile-picture") %>
```
Comment (YilKanda) [inline EasyLearn/app/views/player/index.html.erb]:
```suggestion
						<%= image_tag("user.png", alt: "rss feed", class: "profile-picture") %>
```

File: EasyLearn/test/controllers/player_controller_test.rb [hunk 12]
```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class PlayerControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get player_index_url
```
Comment (YilKanda) [inline EasyLearn/test/controllers/player_controller_test.rb]:
```suggestion
    get player_url
```

File: EasyLearn/app/views/player/index.html.erb (hunk 6 above)
Comment (javs1) [reply EasyLearn/app/views/player/index.html.erb]: Done

File: EasyLearn/app/views/player/index.html.erb (hunk 7 above)
Comment (javs1) [reply EasyLearn/app/views/player/index.html.erb]: Done

File: EasyLearn/app/views/player/index.html.erb (hunk 5 above)
Comment (javs1) [reply EasyLearn/app/views/player/index.html.erb]: Done

File: EasyLearn/app/views/player/index.html.erb (hunk 8 above)
Comment (javs1) [reply EasyLearn/app/views/player/index.html.erb]: Done

File: EasyLearn/app/views/player/index.html.erb (hunk 9 above)
Comment (javs1) [reply EasyLearn/app/views/player/index.html.erb]: Done

File: EasyLearn/test/controllers/player_controller_test.rb (hunk 12 above)
Comment (javs1) [reply EasyLearn/test/controllers/player_controller_test.rb]: Done

File: EasyLearn/app/views/player/index.html.erb [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+
+  <nav>  
+    <h3 >Logo</h3>
+    <div class="menu">
+      <p>My events</p>
+      <div class="user-option">
+        <p class="username">username</p>
+      </div>  
+    </div>
+  </nav>
```
Comment (xFerchoVJ) [inline EasyLearn/app/views/player/index.html.erb]: I think i saw this nav in the past PR of @Joshsam31, and he made an partial of it, you could change this code and use that partial.
Send me a message if you have any questions🤓

File: EasyLearn/app/assets/stylesheets/player.css [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+  align-items: center;
+  background-color:#282828;
+  border-bottom: #F78528 2px solid;
+  box-sizing: border-box;
+  color: white;
+  display: flex;
+  height: 56px; 
+  justify-content: space-between;   
+  padding: 10px 20px;
+}
```
Comment (YilKanda) [inline EasyLearn/app/assets/stylesheets/player.css]:
```suggestion
nav {
  align-items: center;
  background-color:#282828;
  border-bottom: #F78528 2px solid;
  box-sizing: border-box;
  color: white;
  display: flex;
  height: 56px; 
  justify-content: space-between;
  margin-bottom: 3rem;
  padding: 10px 20px;
}
```

File: EasyLearn/app/views/player/index.html.erb (hunk 13 above)
Comment (YilKanda) [reply EasyLearn/app/views/player/index.html.erb]: Totally agree with you @xFerchoVJ

File: EasyLearn/app/views/player/index.html.erb [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+					</div>
+				</div>
+				<div class="extra-video">
+					<video src="" controls> </video>
+					<div class="user">
+						<%= image_tag("user.png", alt: "rss feed", class: "profile-picture") %>
+						<p>Pedro Sanchez</p>
+					</div>
+				</div>
+    	</div>
```
Comment (YilKanda) [inline EasyLearn/app/views/player/index.html.erb:42]: Check the indentation of this part of your code (remember that the spacing must be 2)

File: EasyLearn/app/views/player/index.html.erb (hunk 13 above)
Comment (javs1) [reply EasyLearn/app/views/player/index.html.erb]: Done, changed the navbar

File: EasyLearn/app/assets/stylesheets/player.css (hunk 14 above)
Comment (javs1) [reply EasyLearn/app/assets/stylesheets/player.css]: Changed styles of navbar

File: EasyLearn/app/views/player/index.html.erb (hunk 15 above)
Comment (javs1) [reply EasyLearn/app/views/player/index.html.erb:42]: Fixed identation

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by xFerchoVJ — APPROVED** [review]
LGTM

**Pull request merged**